Extension/package.json

```diff
@@ -593,21 +593,6 @@
         "title": "%c_cpp.command.resetDatabase.title%",
         "category": "C/C++"
       },
-      {
-        "command": "C_Cpp.PauseParsing",
```

Comment:
why don't we need these anymore?

Reply:
We never needed them. They command is usable via clicking the icon.